### ISSUE TITLE
DIVER: Consistent verb use in the cli

### ISSUE THREAD

**Issue by thebsdbox** [joeabbey/diver#35]
- `get`/`set`
- `list` / `ls`
- `create` / `delete`

Need to make a decision on which is the correct usage.

**Comment by johnharris85** [comment]
What about mirroring existing Docker conventions?

`inspect` for getting a resource
`ls` for listing
`rm` to delete
`create` to create

'set' would be implicit like `diver ucp interlock --enable`

Not sure about downloads (like client bundle and potentially support dump), `download`? `pull` (confusing as used for images mostly)? `diver ucp client-bundle download`? Seems fine?

**Comment by thebsdbox** [comment]
Yeah, I'm still up in the air about naming as most of the container ecosystem tools use different verbs.

Docker = `inspect` / Kubectl = `describe` etc..

I'd started to standardise of `create/delete/list/get/set` but I'm happy to change to whatever people think is best :)
